requirements.txt

```diff
@@ -1,13 +1,17 @@
 git+https://github.com/VIP-LES/EosLib@v1.0.0#egg=EosLib
 attrs==22.1.0
 colorama==0.4.5
+exceptiongroup==1.1.0
+greenlet==2.0.2
 iniconfig==1.1.1
 packaging==21.3
 pluggy==1.0.0
 pyparsing==3.0.9
 pytest==7.2.1
 tomli==2.0.1
 digi-xbee==1.4.1
+psycopg2==2.9.5
```

Comment:
We should use either `psycopg2` or `psycopg2-binary`, I see having both in our dependencies causing issues

Reply:
(My vote is for `psycopg2-binary` but I can figure out how to build the other one if I have to)